hack/update-model.sh

```diff
@@ -16,6 +39,9 @@ MODULES=(
 )
 
 echo "Updating Go modules..."
+if [ -n "$COMMIT_SHA" ]; then
+  echo "Using specific commit SHA: $COMMIT_SHA"
```

Comment:
@nimrodshn can you add a sanity check that the commit sha is legit? i.e. it actually refers to an existing git sha1